src/pages/Pokemon/Nav.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './pokemon.css'
+
+const Nav = ({ pokemonItem }) => {
```

Comment:
I know this is a dummy component but we probs don't wanna pass `pokemonItem` here though mate